collectors/metrics/pkg/status/status.go

```diff
@@ -7,39 +7,35 @@ package status
 import (
 	"context"
 	"errors"
-	"fmt"
+	"log/slog"
```

Comment:
The collector and operator are using different loggers. go-kit logger can be adapted to the logr.Logger interface, but it requires yet another dependency.
So we have two solutions:

- either we instantiate a logr logger
- or we can transform to logr easily as logr provides an adaptor. I went for this one, but I don't remember why. Probably because it is in the standard library.

I can implement the other solution if you want. Let me know.